File: stats/views/players/map.js
function(doc) {
	if (doc.type=="game") {
		for (p in doc.teamAstats) {
			emit([doc.teamA,doc.teamAstats[p].name],null)
		}
		for (p in doc.teamBstats) {
			emit([doc.teamB,doc.teamBstats[p].name],null)
		}
	}
}

File: stats/views/individual/map.cs
(doc) ->
  if doc.type=="game" && doc.teamAtuh && doc.teamBtuh
    for player in doc.teamAstats
      pts = ((player.tens*10)-(player.negs*5))
      ppth = pts/player.tuh
      emit([doc.bracket,doc.teamA,player.name],{
        gp:1, tens:player.tens, ppth:ppth, 
        negs:player.negs, tuh:player.tuh, pts:pts, opponent:doc.teamB})

    for player in doc.teamBstats
      pts = ((player.tens*10)-(player.negs*5))
      ppth = pts/player.tuh
      emit([doc.bracket,doc.teamB,player.name],{
        gp:1, tens:player.tens, ppth:ppth,
        negs:player.negs, tuh:player.tuh, pts:pts, opponent:doc.teamA})